app/routes/funding-options/validation.js

```diff
@@ -0,0 +1,30 @@
+const getPollingResponse = require('../../polling')
+const { sendAgreementCalculateMessage } = require('../../messaging')
+const cache = require('../../cache')
+
+module.exports = [{
+  method: 'GET',
+  path: '/funding-options/validation',
+  options: {
+    handler: async (request, h) => {
+      const agreement = await cache.get('agreement', request.yar.id)
+      const response = await getPollingResponse(request.yar.id, '/validate')
+      if (response) {
+        console.info('Validation result received', response)
+        if (response.isValid) {
+          await cache.update('agreement', request.yar.id, { paymentAmount: response.paymentAmount })
+          if (agreement.paymentActions !== undefined && agreement.paymentActions.length > 0) {
+            return h.redirect('land-increased-actions')
+          } else {
+            await sendAgreementCalculateMessage(agreement, request.yar.id)
+            return h.redirect('calculation')
+          }
+        } else {
+          return h.redirect('land-primary-actions')
```

Comment:
As we aren't currently doing the validation and always returning valid, I think this redirect never happens.

Am assuming for now that is fine and not part of the ticket, but just thought I would put a note in to check. If/when we are doing validation we'd want to inform the user of the validation error rather than just redirecting them back to the page with no feedback.

Reply:
Completely agree, we want to redirect the user back to that page, but showing the errors  to the user. As we don't know what the error object will look like at the moment.